src/chrome/content/rules/Fastmail-User.FM.xml

```diff
@@ -0,0 +1,24 @@
+<!--
+          For other Fastmail coverage, see Fastmail.xml 
```

Comment:
Please also add a comment in `Fastmail.xml` pointing to this ruleset.

Reply:
Does this added comment clear it up?

"Fastmail user websites
		If www.example.com was hosted on Fastmail, it would also be available 
		as http://www.example.com.user.fm/ and https://user.fm/www.example.com/ "